src/CalculatorUITestFramework/StandardOperatorsPanel.cs

```diff
@@ -27,5 +26,8 @@ public class StandardOperatorsPanel
         public WindowsElement ClearEntryButton => this.session.TryFindElementByAccessibilityId("clearEntryButton");
         public WindowsElement ClearButton => this.session.TryFindElementByAccessibilityId("clearButton");
         public WindowsElement BackSpaceButton => this.session.TryFindElementByAccessibilityId("backSpaceButton");
+        public WindowsElement NegateButton => this.session.TryFindElementByAccessibilityId("negateButton");
+        public WindowsElement HistoryButton => this.session.TryFindElementByAccessibilityId("HistoryButton");
```

Comment:
History button isn't an operator, so it should be in a different class.

Reply:
Change will be in my next commit